linter.py

```diff
@@ -24,31 +26,33 @@ def find_errors(self, output):
         """
         Parse errors from linter's output.
 
-        We override this method to handle parsing stylelint crashes.
+        We override this method to handle parsing stylelint crashes,
+        deprecations and other feedback about the config.
         """
         data = None
         match = self.crash_regex.match(output)
 
         if match:
             msg = "Stylelint crashed: %s" % match.group(1)
-            yield (match, 0, None, "Error", "", msg, None)
+            self.notify_failure()
+            yield (match, 0, None, "", "Warning", msg, None)
```

Comment:
Do not yield a 'problem'. Call `self.notify_failure(); logger.error(output); return`. `logger.error` for inda face or `logger.warning` just to log it. (So the user has to open the Sublime console.)